packages/extract/src/Extract.ts

```diff
@@ -30,6 +30,15 @@ const BYTES_PER_PIXEL = 4;
  *
  * @memberof PIXI
  */
+
+export interface PixelExtractOptions {
+    x: number,
+    y: number,
+    height: number,
+    resolution: number,
+    width: number
+}
+
```

Comment:
Can you please move this above the comment above? It's breaking the documentation for this Extract class. 

Also, it would be helpful if you could add comments to PixelExtractOptions. Are all fields required?

Reply:
I add comments , I am looking forward your feedback.